pkg/cli/set/env.go

```diff
@@ -50,7 +50,7 @@ var (
 
 	envExample = templates.Examples(`
 		# Update deployment 'registry' with a new environment variable
-	  %[1]s env dc/registry STORAGE_DIR=/local
+	  %[1]s env deployment/image-registry STORAGE_DIR=/local
```

Comment:
dc is not the same as deployment, so you can't change like that. I think we should have examples covering both dc and deployments, still

Reply:
ok, switched back to dc, but modified to `dc/myapp` since `dc/registry` will cause confusion with users, imo.  3.x we deployed a dc/registry, 4.x we don't, (it's deployment/image-registry).